config/routes.rb

```diff
@@ -3,7 +3,9 @@
     resources :items, except: [:destroy]
     resources :categories
   end
-  resource :checkout, only: [:create, :show]
+  resource :checkout, only: [:create, :update]
+
+  get 'checkout', to: "checkouts#edit"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.